### PR TITLE
RHTAPSRE-404: Update build-service to use auto generated ca bundle

### PR DESCRIPTION
**Pull request by gbenhaim**
Openshift will inject the ca bundle data into the trusted-ca configmap. Mount this configmap into the build-service pod.

### CODE REVIEW

File: components/build-service/base/add-ca-bundle-patch.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (MartinBasti) [inline components/build-service/base/add-ca-bundle-patch.yaml:1]: Why these files have been added to base? If it's intentional, it missed entries in kustomize

Comment (gbenhaim) [reply components/build-service/base/add-ca-bundle-patch.yaml]: mistake. I removed them.

File: components/build-service/components/rh-certs/add-rh-certs-patch.yaml [hunk 2]
```diff
@@ -1,30 +1,20 @@
----
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: build-service-controller-manager
+  name: controller-manager
```
[the remark this answers is not in the document]
Comment (gbenhaim) [reply components/build-service/components/rh-certs/add-rh-certs-patch.yaml]: mistake, I removed this change.

**Comment by gbenhaim** [comment]
thanks for the review @MartinBasti

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**